core/src/service/import_export_service.rs

```diff
@@ -80,13 +80,13 @@ fn import_file_recursively(
             fs::read_dir(disk_path).map_err(CoreError::from)?.collect();
 
         if children.is_empty() {
-            path_service::create_at_path(config, &lockbook_path_with_new).map_err(|err| {
-                if err == CoreError::PathTaken {
-                    CoreError::ImportCollision(lockbook_path_with_new)
-                } else {
-                    err
+            match path_service::create_at_path(config, &lockbook_path_with_new) {
```

Comment:
I don't understand the change here, why not just match within the map_err?

Reply:
The `map_err` that was there previously: https://github.com/lockbook/lockbook/pull/856#discussion_r693281304

Reply:
Oh, I thought you meant match the entire thing instead of map_err. I was confused too, but now it makes sense haha.